test/upgrades/upgrade_test.go

```diff
@@ -0,0 +1,76 @@
+package upgrades
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/module-ci/test/upgrades"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+)
+
+// The following lists are to keep track of which of the examples we've added upgrade tests for,
+// and which modules we've tested upgrading. Commented ones are not covered in upgrade tests yet.
+var examplesToTest = upgrades.ExampleConfig{
+	//"executable-dependency",
+	"instance-type": {
+		SetupFn: setupForInstanceType,
+	},
+
+	"join-path": {
+		SetupFn: setupForJoinPath,
+	},
+	//"list-remove",
+	//"operating-system",
+	//"pex",
+	//"request-quota-increase",
```

Comment:
@marinalimeira So when you write a new upgrade test, you only have to uncomment this here, and any modules used by the example, in `modulesToTest` below this.

Then below that, add a `setupForXXX` function that mimics whatever you need for the example to be deployed. If there's anything to teardown after that, you can add a `TeardownFn` underneath the `SetupFn: setupForXXX,` line.